src/main/java/gui/components/BTDialog.java

```diff
@@ -11,9 +19,14 @@
 
 	private static final long serialVersionUID = 1L;
 
-	public BTDialog( final MainFrame frame ) {
+	public BTDialog( final MainFrame frame , String description) {
 		super( frame );
 		this.init();
+		this.setLayout( new BorderLayout() );
+		this.add( this.getMessage(description), BorderLayout.CENTER );
+		this.add( this.getButton(), BorderLayout.SOUTH );
+		
+		this.output();
 	}
 	
 	public BTDialog() {
```

Comment:
Ver se este construtor sem parâmetros é usado em algum lugar, se for, ele não tem a chamada do método `output`, senão remove ele.